packages/swapper/src/swappers/ArbitrumBridgeSwapper/getTradeQuote/getTradeQuote.ts

```diff
@@ -78,14 +78,15 @@ export async function getTradeQuote(
 
   try {
     const swap = await fetchArbitrumBridgeSwap({
-      supportsEIP1559,
+      supportsEIP1559: Boolean(supportsEIP1559),
       chainId,
       buyAsset,
       sellAmountIncludingProtocolFeesCryptoBaseUnit,
       sellAsset,
       sendAddress: sendAddress ?? '',
       receiveAddress,
       assertGetEvmChainAdapter,
+      priceOrQuote: 'price',
```

Comment:
just making a note that if we have a wallet (ie. send/receive address), this type would actually reflect `quote` based on the type definition. also price is broken for arbitrum bridge as we cannot estimate gas without an address. suggestion to either omit gas estimation to show the 1:1 rate, or leverage the placeholder estimation address strategy we took at one point.

Reply:
Added more safety as well as price vs. quote when a wallet is connected in f8ef5825bd

Reply:
https://github.com/shapeshift/web/issues/7976 for follow-up